base/array.jl

```diff
@@ -71,8 +71,27 @@ Union type of [`Vector{T}`](@ref) and [`Matrix{T}`](@ref).
 """
 const VecOrMat{T} = Union{Vector{T}, Matrix{T}}
 
+"""
+    DenseVector{T}
+
+One-dimensional dense array with elements of type `T`, often used to represent
+a mathematical vector. Alias for `DenseArray{T,1}`. The elements of a dense array
+are stored contiguously in memory, with no striding.
```

Comment:
I get what you're trying to say here, but I think I'd refrain from using the phrase "no striding." A contiguous array still has `strides` — it's just that they're arranged such that they cover the memory contiguously with no overlaps or skips.